docs/prerequisites.md

```diff
@@ -0,0 +1,27 @@
+#Prerequisites
+
+##Windows
+
+Ensure you have the following installed:  
+Must Have:  
+[Node.js x64](https://nodejs.org/en/)  
+[Python 2.7](https://www.python.org/downloads/)  
+[Visual Studio 2013 or 2015](https://www.visualstudio.com/en-us/products/visual-studio-community-vs.aspx)  
+
+Should Have:
+
+[Git](https://git-scm.com/download/win)
```

Comment:
Git and node are pretty much always implied.  I'm fine with keeping them though but please make the whole list point form and remove the text `Should Have:`